### PR TITLE
added libsigsegv

### PR DESCRIPTION
**Pull request by jan-janssen**
I tried to build bohrium https://github.com/conda-forge/staged-recipes/pull/5845 , but there seems to be some incompatibility issue with the boost library. Therefore I though I at least submit the libsigsegv as it might be useful for others as well.

### CODE REVIEW

File: recipes/libsigsegv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
```
Comment (mariusvniekerk) [inline recipes/libsigsegv/meta.yaml]: pretty sure we don't need python for this?

File: recipes/libsigsegv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
```
Comment (mariusvniekerk) [inline recipes/libsigsegv/meta.yaml]: You may want to add a 
```
run_exports: 
- {{ pin_subpackage('libsigsegv', max_pin='x.x') }}
```

File: recipes/libsigsegv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - echo "tests are executed during build!"
```
Comment (mariusvniekerk) [inline recipes/libsigsegv/meta.yaml]: can you add
```
    - conda inspect linkages {{ name }}  # [not win]
    - conda inspect objects {{ name }}  # [osx]
```

**Comment by jan-janssen** [comment]
@mariusvniekerk  Thank you for your feedback, I included the suggested changes.

**Pull request merged**